.eslintrc

```diff
@@ -1,6 +0,0 @@
-{
-  "extends": ["frost-standard"],
-  "globals": {
-    "URL": false
```

Comment:
Determine whether the global `URL` value needs to be added to the _.eslintrc.js_ file